Library/Homebrew/software_spec.rb

```diff
@@ -327,16 +327,15 @@ def select_download_strategy(specs)
 
 class BottleSpecification
   DEFAULT_PREFIX = Homebrew::DEFAULT_PREFIX
-  DEFAULT_CELLAR = "#{DEFAULT_PREFIX}/Cellar".freeze
```

Comment:
I don't believe that `BottleSpecification::DEFAULT_PREFIX` is used anywhere else, so it may make sense to just drop this.

Reply:
`Linuxbrew/core` is currently using the idiom `HOMEBREW_PREFIX == BottleSpecification::DEFAULT_PREFIX` where Homebrew has been using `HOMEBREW_PREFIX == "/usr/local"`. So for compatibility this constant needs to remain for now. It could possibly be moved to `compat`.

See https://github.com/Homebrew/brew/pull/4851#discussion_r216400554